File: Back-End/MoviesServer/Models/Movie.cs
using static System.Net.Mime.MediaTypeNames;
using System.IO;

namespace MoviesServer.Models
{
    public class Movie
    {
        public int MovieID { get; set; }
        public string Title { get; set; }
        public int ReleaseYear { get; set; }
        public string Genre { get; set; }
        public decimal Rating { get; set; }
        public int Runtime { get; set; }
        public string Description { get; set; }
        public List<string> Responses { get; set; }
        public string Image { get; set; }

        public Movie(int movieID, string title, int releaseYear, string genre, decimal rating, int runtime, string description, List<string> responses, string image)
        {
            MovieID = movieID;
            Title = title;
            ReleaseYear = releaseYear;
            Genre = genre;
            Rating = rating;
            Runtime = runtime;
            Description = description;
            Responses = responses;
            Image = image;
        }
    }
}

File: Front-End/Entity/movie.py
class Movie:
    def __init__(self, movieID, title, release_year, genre, rating, runtime, description, responses, image):
        self.movieID = movieID
        self.title = title
        self.release_year = release_year
        self.genre = genre
        self.rating = rating
        self.runtime = runtime 
        self.description = description
        self.responses = responses
        self.image = image

    def __str__(self):
        return f"{self.title} ({self.release_year}), Genre: {self.genre}, Rating: {self.rating}/10"
